wikichange/src/content/content.js

```diff
@@ -35,21 +35,46 @@ const renderGraphOverlay = async () => {
 
     let graphContainer = document.createElement("div");
     graphContainer.setAttribute("id", "graphOverlay");
-
+    graphContainer.setAttribute("text-align", "center");
     let canvas = document.createElement("canvas");
     canvas.style.maxHeight = "200px";
     canvas.id = "viewsEditsChart";
-    graphContainer.style.cssText = "width:75%;height:20%;";
+    graphContainer.style.cssText = "width:100%;height:20%;";
     graphContainer.appendChild(canvas);
-
     floatContainer.appendChild(graphContainer);
 
     let p = document.createElement("p");
     graphContainer.appendChild(p);
 
+    const ctx = canvas.getContext('2d');
+    let percentage = 0;
+    let diff;
+    
+    function progressBar() {
```

Comment:
Can you place the value of the progress bar to be in the center of the circle? The 100% doesn't look good

Reply:
And please, use the same blue color as the slider or the highlight loader. Let's keep it consistent

Reply:
Yup. seeing the same thing. the percent should be in the center of the circle and the color should match the old highlight. Otherwise, it looks good.
<img width="306" alt="image" src="https://user-images.githubusercontent.com/55929961/213617809-5ad92370-102c-4a69-95d0-534b15e064ac.png">

Reply:
I just check and I am also surprised that this is the same blue color as the graph. The slider set a default blue color so I can't get the correct value for the color but I think it is a brighter blue. What do you guys think the color should be? @sukritsangvong @LuisaE

Reply:
I think she was referring to the loader of the slider. I can see that the shade of blue is a little bit different. If that's not the case, then u don't have to change.
![image](https://user-images.githubusercontent.com/55929961/213812225-5de02189-a8e4-43e0-845e-926df338a219.png)